images/win/scripts/Installers/Install-Pipx-Packages.ps1

```diff
@@ -0,0 +1,12 @@
+$pipxToolset = (Get-ToolsetContent).pipx
+foreach($tool in $pipxToolset) {
+    if ($tool.python) {
+        $pythonPath =  "${env:AGENT_TOOLSDIRECTORY}\Python\${tool.python}*\${env:AGENT_OSARCHITECTURE}\python-${tool.python}*"

```

Comment:
1) `$pythonPath` vs `$python_path`
2) `"${env:AGENT_TOOLSDIRECTORY}\Python\${tool.python}.*\x64\python-${tool.python}.*"`
3) You need `Get-Item` to get exact path